lib/AlmaClient/AlmaClient.class.php

```diff
@@ -544,7 +544,9 @@ public function renew_loan($borr_card, $pin_code, $loan_ids) {
           // successful. Even if this is not the case any error in the current
           // renewal is irrelevant as the loan has previously been renewed so
           // don't report it as such.
-          if ($message == 'isRenewedToday' || $renewable == 'yes') {
+          // Also renewalIsDenied marked as 'no' is considered a successful
+          // renewal.
```

Comment:
Thanks for elaborating. Perhaps we could add that this "renewalIsDenied=no" is relevant for interlibrary loans.